### PR TITLE
CSS: Improve links contrast on dark theme

### LINKED ISSUE #672: Footer font color is hard to read when using Dark Mode
In my opinion the color of the font used in the footer is a bit too dark when using dark mode, I don't personally have a problem to read it but I guess it might be better to have a lighter color for people with vision problems.

White would be too "visible" so I guess using a color a bit darker than the one used for the "normal" text (#a0a0a0) would be the way to go.

footer

### PR DESCRIPTION
**Pull request by RadoslavL**
This closes #672.

### CODE REVIEW

**Review by syeopite — APPROVED** [review]
Looks good to me. But why did you make a new PR?

**Comment by RadoslavL** [comment]
> Looks good to me. But why did you make a new PR?

In my first PR I didn't create a seperate branch, so I couldn't delete it after the PR is merged.

**Comment by SamantazFox** [comment]
It needs testing, as it's impacting all links, not only titles.

Also, you'll need to change the value here, too (the one you changed is for the forced dark theme, and the one I've linked below is for the automatic dark theme, based on browser/system settings):
https://github.com/iv-org/invidious/blob/beec62cf0e45fe5620f9381050080c685f32070e/assets/css/default.css#L637-L640

**Comment by RadoslavL** [comment]
I noticed that the footer was unaffected by this change, as it is set by `footer a` instead of `.dark-theme a`, so changing it would affect both dark and light theme. I think I can work around that.

Before:
screenshot_before

After:
screenshot_after

**Comment by RadoslavL** [comment]
I completely misunderstood #672. I will still try to solve it.

**Comment by RadoslavL** [comment]
I changed the footer color in dark theme from `#919191` to `#adadad`, and in light theme from `#919191` to `#7c7c7c`.

Light theme:

Before:
screenshot2_before

After:
screenshot2_after

Dark theme:

Before:
screenshot1_before

After:
screenshot1_after

File: assets/css/default.css [hunk 1]
```diff
@@ -432,17 +432,30 @@ p.video-data   { margin: 0; font-weight: bold; font-size: 80%; }
...
  */
 
-footer {
-  color: #919191;
+.light-theme footer {
+  color: #7c7c7c;
   margin-top: auto;
   padding: 1.5em 0;
   text-align: center;
   max-height: 30vh;
```
Comment (syeopite) [inline assets/css/default.css:440]: There's no need for you to duplicate these styles so many times across the file. You can just place them under a footer element selector once.

File: assets/css/default.css [hunk 2]
```diff
@@ -433,16 +433,28 @@ p.video-data   { margin: 0; font-weight: bold; font-size: 80%; }
...
+  color: #7c7c7c;
+}
+
+.dark-theme footer {
+  color: #adadad;
+}
+
+.light-theme footer a {
+  color: #7c7c7c !important;
+/*text-decoration: underline;*/
```
Comment (SamantazFox) [inline assets/css/default.css:452]: Can you remove the commented lines please?

**Comment by SamantazFox** [comment]
Thanks for your contribution :D

**Pull request merged**